vllm/reasoning/abs_reasoning_parsers.py

```diff
@@ -77,7 +77,7 @@ def extract_reasoning_content(
         self,
         model_output: str,
         request: Union[ChatCompletionRequest, ResponsesRequest],
-    ) -> tuple[Optional[str], Optional[str]]:
+    ) -> tuple[str | None, str | None]:
```

Comment:
According to [https://docs.vllm.ai/en/latest/getting\_started/quickstart.html#prerequisites](https://docs.vllm.ai/en/latest/getting_started/quickstart.html#prerequisites), vLLM still supports Python 3.9, so you shouldn’t use this syntax.